From ce8f1b697fb64e839464757fafe5afde6743486d Mon Sep 17 00:00:00 2001
From: Patryk Kups <pkups@anaconda.com>
Date: Mon, 26 Jan 2026 09:59:21 +0100
Subject: [PATCH] fix hardcoded version

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 39a6c09..4215760 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "flask-cors"
-version = "0.0.1"
+version = "6.0.2"
 description = "A Flask extension simplifying CORS support"
 authors = [{ name = "Cory Dolphin", email = "corydolphin@gmail.com" }]
 readme = "README.rst"
-- 
2.39.5 (Apple Git-154)


